src/test/java/edu/cornell/kfs/module/purap/jaggaer/supplier/xml/SupplierSyncMessageCornellTest.java

```diff
@@ -54,6 +67,40 @@ static Stream<Arguments> testSupplierSyncMessageArguments() {
                 Arguments.of(RESPONSE_GOOD_FILE_EXAMPLE, false, true),
                 Arguments.of(RESPONSE_BAD_FILE_EXAMPLE, false, false));
     }
+    
+    @Test
+    void TestResponseMessageString() throws JAXBException, SAXNotRecognizedException, SAXNotSupportedException, ParserConfigurationException, XMLStreamException, IOException {
+        String xmlString = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+                + "<!DOCTYPE SupplierSyncMessage SYSTEM \"https://usertest-messages.sciquest.com/app_docs/dtd/supplier/TSMSupplier.dtd\">\n"
+                + "<SupplierSyncMessage version=\"1.0\">\n"
+                + "    <Header>\n"
+                + "        <MessageId>\n"
+                + "            fb95040f-ee74-43ae-b38d-01898cd28033\n"
+                + "        </MessageId>\n"
+                + "        <Timestamp>\n"
+                + "            2023-07-25T07:32:32.947-04:00\n"
+                + "        </Timestamp>\n"
+                + "    </Header>\n"
+                + "    <SupplierResponseMessage>\n"
+                + "        <Status>\n"
+                + "            <StatusCode>\n" 
+                + "                " + STATUS_CODE_200 + "\n"
+                + "            </StatusCode>\n"
+                + "            <StatusText>\n"
+                + "                " + STATUS_TEXT + "\n"
+                + "            </StatusText>\n"
+                + "        </Status>\n"
+                + "\n"
+                + "    </SupplierResponseMessage>\n"
+                + "</SupplierSyncMessage>";
+        
+        SupplierSyncMessage synchMessage = marshalService.unmarshalStringIgnoreDtd(xmlString, SupplierSyncMessage.class);
```

Comment:
Should this variable be renamed to `syncMessage`?

Reply:
ok